kernel/os/include/os/os_eventq.h

```diff
@@ -71,11 +71,14 @@ struct os_eventq {
      */
     struct os_task *evq_task;
 
+#if MYNEWT_VAL(OS_EVENTQ_DEBUG)
+    /** Most reently procesed event. */
```

Comment:
typo `reently`

Reply:
btw, `procesed` is also a typo!

Reply:
That's pretty funny :).  Thanks for catching those.  I have fixed and force pushed.